plugin/storage/es/spanstore/reader.go

```diff
@@ -129,10 +129,17 @@ type SpanReaderParams struct {
 
 // NewSpanReader returns a new SpanReader with a metrics.
 func NewSpanReader(p SpanReaderParams) *SpanReader {
+	maxSpanAge := p.MaxSpanAge
+	// If rollover is enabled set lookback far into the past
+	// In rollover only read alias is used to query the data so looking far into the past should
+	// not affect performance.
```

Comment:
I suggest we reword this to describe why `maxSpanAge` is set to a large number. A suggestion:
```
Setting the maxSpanAge to a large duration will ensure all spans in the "read" alias are accessible by queries (query window = now - maxSpanAge).
When read/write aliases are enabled, required for index rollovers, only the "read" alias is queried and therefore should not affect performance.
```